# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/process-cpp/hotfix-cmake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- process-cpp-2923b597f6fc5b49133be8c4f2ba3cbaacdb9540/CMakeLists.txt.vanilla2026-08-08 19:14:39.412603445 +0300
+++ process-cpp-2923b597f6fc5b49133be8c4f2ba3cbaacdb9540/CMakeLists.txt	2026-08-08 19:14:47.444388216 +0300
@@ -18,7 +18,7 @@
 
 project(process-cpp)
 
-find_package(Boost COMPONENTS iostreams system REQUIRED)
+find_package(Boost COMPONENTS iostreams REQUIRED)
 find_package(PkgConfig REQUIRED)
 find_package(Threads REQUIRED)
 
@@ -73,4 +73,4 @@
 add_subdirectory(data)
 add_subdirectory(include)
 add_subdirectory(src)
-add_subdirectory(tests)
+#add_subdirectory(tests)

